src/breaker.js

```diff
@@ -8,7 +8,16 @@ const {tween} = require('./utils.js');
 
 /* Methods -------------------------------------------------------------------*/
 
-function breaker(config, emitter) {
+function normalizeConfig(config){
+    if(!config){
+        config = {};
+    }
+
+    return config;
+}
+
+function breaker(haStoreConfig = {}, emitter) {
+  const config = normalizeConfig(haStoreConfig);
```

Comment:
why not default it in the params or using pipes to initiate it ex: `config = config || {};`  ?

Reply:
Its part `it made sense in my first implementation` and `id like to keep the configuration out of the bl`

But yeah, its pretty much useless atm.